tests/e2e/Services/Users/UsersBase.php

```diff
@@ -644,6 +644,17 @@ public function testGetUser(array $data): array
         $this->assertIsInt($users['body']['total']);
         $this->assertGreaterThan(0, $users['body']['total']);
 
+        
+        /** 
+          * Test for FAILURE 
+          */ 
+         $response = $this->client->call(Client::METHOD_GET, '/users' . $data['userId'], array_merge([ 
```

Comment:
The ID passed in here should be for an ID that doesn't exist so that it returns 404.